config/jobs/kubernetes/test-infra/test-infra-trusted.yaml

```diff
@@ -1031,6 +1031,28 @@ periodics:
     testgrid-tab-name: label_sync
     testgrid-alert-email: kubernetes-sig-testing-alerts@googlegroups.com, k8s-infra-oncall@google.com
     description: Runs label_sync to synchronize GitHub repo labels with the label config defined in label_sync/labels.yaml.
+- name: test-infra-merge-testgrid-config-canary
+  interval: 30m
+  cluster: test-infra-trusted
+  max_concurrency: 1
+  decorate: true
+  spec:
+    serviceAccountName: testgrid-config-updater
+    containers:
+    - image: gcr.io/k8s-testgrid/config_merger:latest
```

Comment:
Can we please pin an actual version, rather than a floating tag like latest?

Reply:
Fixed in TG review